src/main.cpp

```diff
@@ -306,6 +306,10 @@ static void parse_env(BPFtrace& bpftrace)
     config_setter.set(ConfigKeyInt::max_cat_bytes, x);
   });
 
+  get_bool_env_var("BPFTRACE_PRINT_MAPS_ON_EXIT", [&](bool x) {
+    config_setter.set(ConfigKeyBool::print_maps_on_exit, x);
+  });
+
```

Comment:
I view this config as a property of the source code, not the environment it's run in. ie. should only be configurable using `config = ` block. So I'd vote for omitting the env var here.

Reply:
Funny, I had the opposite view because it actually doesn't affect the probes at all, and just affects what the tool does at the end. (I first considered just adding as a command line flag, which is a further step in that direction.) Either way, I sort of feel like it's good to be consistent and just have these settable uniformly via environment variables? If you feel strongly I can remove this.

Reply:
I kind of agree with @amscanne though I don't feel strongly. It seems like a property or preference for whoever is running the script. That being said I know we're trying to not add more ENV vars... so 🤷

Reply:
Not relying on env variables is definitely a good thing IMHO, but personally I value consistency over most other things :) If any decider wants to call it, I'm happy to do whatever with this, I just wanted to note the differences in perspective first

Reply:
I'm kinda looking at it from the view of pre-written scripts that rely on the default printing behavior to display output to user. In that case I think config only makes sense. I think there's already more config vars than env vars (can double check me). FWIW I don't have a strong preference either, probably fine to go as-is. If so, pls update the `-h` message. At some point we should think about culling the list or pointing to the docs.

Reply:
I've dropped the environment variable. Even the use is ambiguous, better to not allow something that folks might depend on. It can always be revisited later.